.gitignore

```diff
@@ -25,3 +25,5 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+public/.DS_Store
+.DS_Store
```

Comment:
I wish Github would just include this in all of their .gitignore boilerplates :rage: